framework/doc/content/source/actions/CreateApplicationBlockAction.md

```diff
@@ -0,0 +1,9 @@
+# CreateApplicationBlockAction
+
+!syntax description /Application/CreateApplicationBlockAction
+
+This action is parse the parameters passed to the `[Application]` block.
```

Comment:
`parses`

but I thought we were not doing this through an action in the end

Reply:
I think for this PR  we decide to use action, right  @loganharbour?  I'll do another PR to create a application system and maybe parse the parameters inside Appfactory

Reply:
It's there